stable_baselines3/common/logger.py

```diff
@@ -140,6 +140,24 @@ def close(self) -> None:
             self.file.close()
 
 
+def filter_excluded_keys(
+    key_values: Dict[str, Any], key_excluded: Dict[str, Union[str, Tuple[str, ...]]], _format: str
+) -> Dict[str, Any]:
+    """
+    Filters the keys specified by ``key_exclude`` for the specified format
+
+    :param key_values: log dictionary to be filtered
+    :param key_excluded: keys to be excluded per format
+    :param _format: format for which this filter is run
+    :return: dict without the excluded keys
+    """
+
+    def is_excluded(key: str):
```

Comment:
```suggestion
    def is_excluded(key: str) -> bool:
```

Reply:
Darn, sry that one slipped through again - not used to these python type hints ;)